telco-examples/edge-clusters/dhcp-less/edge-telco-single-node/telco-capi-single-node-sriov-auto.yaml

```diff
@@ -88,6 +88,34 @@ spec:
                 name: root
               group:
                 name: root
+            - path: /var/lib/rancher/rke2/server/manifests/sriov-crd.yaml
+              overwrite: true
+              contents:
+                inline: |
+                  apiVersion: helm.cattle.io/v1
+                  kind: HelmChart
+                  metadata:
+                    name: sriov-crd
+                    namespace: kube-system
+                  spec:
+                    chart: oci://registry.suse.com/edge/charts/sriov-crd-chart
+                    targetNamespace: kube-system
+                    version: 1.2.2
+                    createNamespace: true
+            - path: /var/lib/rancher/rke2/server/manifests/sriov-network-operator.yaml
+              overwrite: true
+              contents:
+                inline: |
+                  apiVersion: helm.cattle.io/v1
+                  kind: HelmChart
+                  metadata:
+                    name: sriov-network-operator
+                    namespace: kube-system
+                  spec:
+                    chart: oci://registry.suse.com/edge/charts/sriov-network-operator-chart
+                    targetNamespace: kube-system
```

Comment:
Does this need to use `kube-system`?  The [chart docs](https://github.com/suse-edge/charts/tree/main/charts/sriov-network-operator/1.2.4%2Bup0.1.0#deploy-sr-iov-network-operator) describe using a `sriov-network-operator` namespace which would probably be preferable?